frontend/src/store/index.js

```diff
@@ -64,6 +68,152 @@ export default new Vuex.Store({
 			return user;
 		},
 
+		async getMeApollo({ commit }) {
+			const user = await apolloClient.query({
+				query: gql`
+					query {
+						me {
+							id
+							username
+							fullName
+							email
+						}
+					}
+				`
+			});
+			console.log("user", user.data.me);
+			//commit("SET_LOGGED_IN_USER", user.data.me);
+			return user.data.me;
+		},
+		async getBoard({ commit }, id) {
+			const res = await apolloClient.query({
+				query: gql`
+					query board($id: String!) {
+						board(id: $id) {
+							id
+							title
+							slug
+							description
+							position
+							archived
+							public
+						}
+					}
+				`,
+				variables: { id }
+			});
+			commit("SET_BOARD", res.data.board);
+			console.log("getBoard", res.data.board);
+			return res.data.board;
+		},
+
+		async getBoards({ commit }) {
+			const res = await apolloClient.query({
+				query: gql`
+					query {
+						boards {
+							id
+							title
+							description
+							owner {
+								username
+								fullName
+								boards {
+									title
+								}
+							}
+						}
+					}
+				`
+			});
+			commit("SET_BOARDS", res.data.boards);
+			return res.data.boards;
+		},
+
+		async createBoard({ commit }, input) {
+			// Call to the graphql mutation
+			console.log("input", input);
+			apolloClient
+				.mutate({
+					// Query
+					mutation: gql`
+						mutation createBoard($input: CreateBoardInput!) {
+							createBoard(input: $input) {
+								id
+								title
+								description
+							}
+						}
+					`,
+
+					// Parameters
+					variables: input
+				})
+				.then(async data => {
+					// Result
```

Comment:
Would be good to show a toast/notification with izitoast if the operation was success or failed in create/update/remove actions.